Formula/awscli.rb

```diff
@@ -16,22 +26,100 @@ class Awscli < Formula
     sha256 cellar: :any_skip_relocation, x86_64_linux:  "2f24916fd69ccdb28d634ce361dc33fd7d82cf4d376d1e527b9ec3fe47c12799"
   end
 
+  head do
+    url "https://github.com/aws/aws-cli.git", branch: "v2"
+
+    resource "botocore" do
+      url "https://github.com/boto/botocore.git", branch: "v2"
+    end
+  end
+
   depends_on "cmake" => :build
   depends_on "python@3.9"
+  depends_on "six"
 
   uses_from_macos "groff"
 
   on_linux do
     depends_on "libyaml"
   end
```

Comment:
Note to check on this. `awscli` has dependency on `ruamel-yaml`, which has its own C library implementation. On other hand, the alternative `PyYAML` relies on `libyaml`.

So, seems odd to need `libyaml` here.

Reply:
At least the Python YAML dependency doesn't link to libyaml, so might as well try with it removed.
```console
# objdump -p /home/linuxbrew/.linuxbrew/Cellar/awscli/2.2.28/libexec/lib/python3.9/site-packages/_ruamel_yaml.cpython-39-x86_64-linux-gnu.so | grep NEEDED -A1
  NEEDED               libpthread.so.0
  NEEDED               libc.so.6
  RPATH                /home/linuxbrew/.linuxbrew/Cellar/awscli/2.2.28/lib:/home/linuxbrew/.linuxbrew/lib:/home/linuxbrew/.linuxbrew/opt/gdbm/lib:/home/linuxbrew/.linuxbrew/opt/mpdecimal/lib ...
```